### PR TITLE
docs: add gateway

### PR DESCRIPTION
**Pull request by xiang90**
/cc @gyuho @heyitsanthony

### CODE REVIEW

**Comment by gyuho** [comment]
lgtm. thanks!

File: Documentation/op-guide/clustering.md [hunk 1]
```diff
@@ -456,6 +456,10 @@ $ etcd --name infra2 \
 --listen-peer-urls http://10.0.1.12:2380
 ```
 
+### Gateway
+
+etcd gateway is a simple TCP proxy that forwards network data to the etcd cluster. Please read [gateway] doc for more information.
```
Comment (heyitsanthony) [inline Documentation/op-guide/clustering.md]: s/[gateway] doc/[gateway guide][gateway]/

**Comment by xiang90** [comment]
@heyitsanthony Fixed.

**Comment by heyitsanthony** [comment]
lgtm

**Pull request merged**